packages/domain/lib/usecase/check_number_usecase.dart

```diff
@@ -0,0 +1,15 @@
+import 'usecase.dart';
+
+class CheckNumberParams {
```

Comment:
better to move in to separate folder, models for example

Reply:
corrected